src/analyses/tutorials/constants.ml

```diff
@@ -1,3 +1,4 @@
+(** Simple intraprocedural integer constants analysis example ([constants]). *)
```

Comment:
Maybe add a link to the tutorial?

Reply:
This isn't even a tutorial, but just an example that you guys added at some point.

Reply:
Ah, right.